e2e/deploy-rook.go

```diff
@@ -79,15 +81,52 @@ func deployOperator(c kubernetes.Interface) {
 func deployCluster(c kubernetes.Interface) {
 	opPath := fmt.Sprintf("%s/%s", rookURL, "cluster-test.yaml")
 	framework.RunKubectlOrDie("create", "-f", opPath)
-	err := waitForDaemonSets("rook-ceph-agent", rookNS, c, deployTimeout)
-	Expect(err).Should(BeNil())
+
+	// After rook-ceph v1.1.0, flex driver is disabled. We don't need to wait for rook-ceph-agent.
+	if !strings.EqualFold(RookVersion, "master") && isOrderRookVersionThan(RookVersion, "v1.1.1") {
+		err := waitForDaemonSets("rook-ceph-agent", rookNS, c, deployTimeout)
+		Expect(err).Should(BeNil())
+	}
+
 	opt := &metav1.ListOptions{
 		LabelSelector: "app=rook-ceph-mon",
 	}
-	err = checkCephPods(rookNS, c, 1, deployTimeout, opt)
+	err := checkCephPods(rookNS, c, 1, deployTimeout, opt)
 	Expect(err).Should(BeNil())
 }
 
+func isOrderRookVersionThan(targetVersion, compareToVersion string) bool {
```

Comment:
Did you mean to call this function `isOlderRookVersionThan`?

Reply:
@nixpanic Oh, Thanks. It's my mistake. Change Done!